src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,368 @@
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>]_probseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-BG_probseg.nii.gz
+            sub-001_space-orig_label-WM_probseg.nii.gz
+```
+
+See [Anatomical labels](#anatomical-labels) for reserved key values for `label`.
+
+A 4D probabilistic segmentation, in which each frame corresponds to a different
+tissue class, must provide a label mapping in its JSON sidecar. For example:
+
+```Text
+pipeline/
+    sub-001/
+	    anat/
+		    sub-001_space-orig_probseg.nii.gz
+		    sub-001_space-orig_probseg.json
+```
+
+The JSON sidecar must include the label-map key that specifies a tissue label
+for each volume:
+
+```JSON
+{
+    "LabelMap": [
+        "BG",
+        "WM",
+        "GM"
+        ]
+}
+```
+
+Values of `label` need to map to Abbreviations defined in [Anatomical Labels](#anatomical-labels).
+
+### Surface Parcellations
+
+Discrete parcellations (surface segmentations) of cortical structures should be
+stored as GIFTI or CIFTI file.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_entities>[_hemi-{L|R}][_space-<space>][_res-<label>][_den-<label>]_dseg.{label.gii|dlabel.nii}
+```
+
+The REQUIRED extension for GIFTI parcellations is `.label.gii`. The `hemi` tag is
+REQUIRED for GIFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_dseg.label.gii
+            sub-001_hemi-R_dseg.label.gii
+```
+
+The REQUIRED extension for CIFTI parcellations is `.dlabel.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_dseg.dlabel.nii
+            sub-001_dseg.dlabel.nii
+```
+
+### Anatomical Labels
+
+BIDS supplies a standard, generic label-index mapping, defined in the table
+below, that contains common tissue classes and can be used to map segmentations
+(and parcellations) between lookup tables.
+
+| Integer value | Description             | Abbreviation (label) |
+| ------------- | ----------------------- | -------------------- |
+| 0             | Background              | BG                   |
+| 1             | Gray Matter             | GM                   |
+| 2             | White Matter            | WM                   |
+| 3             | Cerebrospinal Fluid     | CSF                  |
+| 4             | Gray and White Matter   | GWM                  |
+| 5             | Bone                    | B                    |
+| 6             | Soft Tissue             | ST                   |
+| 7             | Non-brain               | NB                   |
+| 8             | Lesion                  | L                    |
+| 9             | Cortical Gray Matter    | CGM                  |
+| 10            | Subcortical Gray Matter | SCGM                 |
```

Comment:
```suggestion
| 10            | Subcortical Gray Matter | SGM                  |
```

Reply:
This works for me... Does anybody have an objection? I don't recall if we were generating these based on some existing table or just trying to make something sensible.

Reply:
In the absence of objection, merging.

Reply:
I'd earlier made a more grand objection here (#270); but I think I lost on that one.